### ISSUE TITLE
node remove should require a node name

### ISSUE THREAD

**Issue by seanjensengrey** [basho-labs/riak-mesos-tools#31]
```
riak-mesos node remove
```

will remove `riak-default-1` by default when no name is provided.

On subsequent, rexecution

```
$ riak-mesos node remove
Resource at http://10.1.12.145:31936/api/v1/clusters/default/nodes/riak-default-1 was not found (Status Code: 404)
```

The command _should_ require an explicit node name, for example

```
riak-mesos node remove --node riak-cluster-4
```

**Comment by sanmiguel** [comment]
You're right, this should be a required argument. In which case, I think we
should promote it to a positional arg, not a switch: `riak-mesos node
remove riak-default-1`

That's 7 less characters to type!

I'm tempted to suggest that this applies to all node commands (except add).

Thoughts?

**Comment by seanjensengrey** [comment]
and if we have a bare

`riak-mesos node remove`

it should actually call 

`riak-mesos node list` along with help output
